mysettings.py

```diff
@@ -18,6 +18,8 @@
 BLOG_URL = 'https://blog.jetbrains.com/kotlin/'
 FORUM_URL = 'https://discuss.kotlinlang.org/'
 BUGTRACKER_URL = 'https://youtrack.jetbrains.com/issues/KT'
+KOTLIN_CONTRIBUTORS = 'https://github.com/JetBrains/kotlin/graphs/contributors'
```

Comment:
Commonly it's used for links used more than one time.

Reply:
Add link in markup.